templates/email/bromley/_council_reference_alert_update.html

```diff
@@ -0,0 +1,3 @@
+[% IF title == 'Garden Subscription - New' %]
+<a href="/waste[% IF property_id %]/[% property_id %][% END %]">Check your bin collections day</a>
```

Comment:
I don't think `property_id` is available at this point? I can't see what would be setting it. You need to look it up in the template from `report`, see how the link I mentioned on the report page does it.